src/System.Private.CoreLib/shared/System/Numerics/Vector.cs

```diff
@@ -778,7 +778,65 @@ private Vector(ref Register existingRegister)
 
 #if netcoreapp
         /// <summary>
-        /// Constructs a vector from the given span. The span must contain at least Vector'T.Count elements.
+        /// Constructs a vector from the given <see cref="ReadOnlySpan{Byte}"/>. The span must contain at least <see cref="Vector{Byte}.Count"/> elements.
+        /// </summary>
+        public Vector(ReadOnlySpan<byte> values)
+            : this()
+        {
+            if ((typeof(T) == typeof(byte))
```

Comment:
nit: Would be nice to extract these into an `IsSupported` check like we did for `Vector128<T>` (and others):
https://source.dot.net/#System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector128_1.cs,76

Reply:
Do you know if this would mess with the codegen? I experimented and didn't see a difference in my limited testing, but I also don't know the full breadth of scenarios where codegen might produce suboptimal results.

Reply:
We are already doing it in the HWIntrinsics, so I don't believe it does.